### PR TITLE
Fix: namespaces

### PR DESCRIPTION
**Pull request by pms1969**
Renames namespace configs so that they are picked up first.

    This prevents kubectl from applying them in the wrong order
    when recursively applying a directory.

    i.e. `kubectl apply -f addons/prometheus/ -R`
    would fail with namespace not available errors.  This should not happen now.

## Testing

deployed all of the addons successfully.

### CODE REVIEW

File: addons/prometheus/_namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```
Comment (dghubble) [inline addons/prometheus/_namespace.yaml]: There is already a namespace in the prometheus directory, with your change there would be two.

Comment (pms1969) [reply addons/prometheus/_namespace.yaml]: yeah, spotted that.  I think there might actually be a case for moving the namespaces out into their own addon?

**Comment by pms1969** [comment]
Will make changes soon, and push changed commit message.

File: addons/prometheus/_namespace.yaml (hunk 1 above)
Comment (dghubble) [reply addons/prometheus/_namespace.yaml]: Doesn't make sense to me that they'd be in a separate addon. Rationale?

**Comment by dghubble** [comment]
Hm? Please squash your commits into one commit anyway.

**Comment by pms1969** [comment]
Done.

**Comment by dghubble** [comment]
Looks good. What about the namespace in nginx-ingress for google-cloud and digital-ocean?

**Comment by pms1969** [comment]
Done.  Sorry, been so focused on getting other stuff working, I hadn't even clocked that those directories existed.

**Comment by dghubble** [comment]
Thanks! Rebased and merged as 86420fd5072799a73b1dd729d5f6b188d71b672f

**Pull request closed without merging**